torch/csrc/jit/python/script_init.cpp

```diff
@@ -783,11 +787,51 @@ void initJitScriptBindings(PyObject* module) {
                   if (auto method = self.find_method(name)) {
                     return py::cast(*method);
                   }
+                  if (self.has_property(name)) {
+                    auto prop = self.get_property(name);
+                    auto getter_func = py::cast(prop.getter_func);
+                    auto setter_func = py::cast(prop.setter_func);
```

Comment:
Hm, how come you are able to call `py::cast` without a template argument? What does the function do in this case?
```suggestion
                    auto getter_func = py::cast(prop.getter_func);
```

Reply:
Otherwise, you get "No matching function for call to object of type 'torch::jit::Method'" error. I think you can use py::cast without template argument to make the method callable python function or something. This pattern was used a lot in script_init.cpp.

Reply:
done